code/index.html

```diff
@@ -3,28 +3,131 @@
+        <nav class="navbar">
+          
+          <a href="#" class="logo">Frusto</a>
+          <ul class="nav-menu">
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">About us</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">News</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Contact</a>
+            </li>
+          
+          </ul>
+          </nav>
+      </div>
+
+      
+      <div class="logo-place">
+        <a href="#" class="logo-mobil">Frusto</a>
+      </div>
+      
+    </header> 
 
+  <main class="main-container">
+    
     <!-- Signup form -->
+    <section class="form-wrapper">
+      <h2>Sign up and start unloading your feelings</h2> 
+      <form id="form" class="form-inputs" action="https://httpbin.org/anything" method="post"> 
+      
+        <div class="textarea">
+          <label for="fname">First name</label>
+          <input id="fname" type="text" name="fname"/>
+        
+          <label for="lname">Last name</label>
+          <input id="lname" type="text" name="lname" />
+
+          <label for="email">email</label>
+          <input id="email" type="email" name="email"/>
+        
+          <label for="password">Password</label>
+          <input id="password" type="password" name="password" required/>
+        </div>
+      
+   
+        <div class="radiobutton">
+          <div>
+            <input id="prof" type="radio" name="occupation" value="prof"/>
+            <label for="prof">young and restless</label>
+          </div>
+        </div>
+        
+        <div>
+          <input id="student" type="radio" name="occupation" value="student"/>
+          <label for="student">Tired and frustrated</label>
+        </div>
+      
+        <div class="checkbox">
+          <input id="agreement" type="checkbox" name="agreement"/>
+          <label for="agreement">Agree to the terms and conditions.</label>
+          <br>
+          <input id="newsletter" type="checkbox" name="news"/>
+          <label for="newsletter">Subscribe to the newsletter.</label>
+        </div>
+      
+        <button id="submit" type="submit" name="submit">SIGNUP NOW</button>
+  
+      </form>
+    </section> 
+
 
-    <section class="accordion">
-      <!-- Accordion -->
+    <!-- Accordion -->
+    <section class="accordion-container">
+    
+      <p>So nice that you have made it here! This is an online community for people who feel they are tired of everything around them including themselves. 
+        The goal with this website is to gather people who want to make a difference for oneself and others.
+        Put simply, this is a place for frustrated people to unload their feelings. </p>
+      <br>
       <h1>FAQ</h1>
```

Comment:
`<br>` is maybe not the best choice, convention wise, for wanting to put some space between your `<p>` tag and your FAQ header. I think I would maybe have added a class to your FAQ header, and added either some margin or padding to the top and bottom to create a bit of space around the FAQ.
By adding a class to the `h1` tag for the FAQ, you could also `text-align: center` it, as well as change the font size to be more similar to the accordion headers.
You could also have added a class to your `<p>` tag, which would have allowed you to add the line-height property, which makes it a bit more legible / adds some space between the lines of text.